### PR TITLE
eu.planete_kraus.Tarot

### PR DESCRIPTION
**Pull request by play-tarot-2**
Hello,
I am starting to write a little app to play an awesome card game, which is quite well known in France and to some extent in Germany. It is still in early development stage, but you can play a game till the end.
I have translated the rules to English https://play-tarot.frama.io/tarot/share/doc/tarot/tarot.html/The-game.html#The-game.
I hope I meet all app requirements, but since this is my first application I may have missed something. Please pardon my ignorance.
There has been a problem with my first github account, so please ignore https://github.com/flathub/flathub/pull/949 if you can see it.

Best regards,

Vivien

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build eu.planete_kraus.Tarot

File: eu.planete_kraus.Tarot.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "runtime": "org.freedesktop.Platform",
+    "sdk": "org.freedesktop.Sdk",
+    "runtime-version": "18.08",
+
+    "command": "tarot",
+    "finish-args": [
+	"--socket=x11",
+	"--socket=wayland",
+	"--share=ipc"
+    ],
```
Comment (barthalion) [inline eu.planete_kraus.Tarot.json]: Please add something like:

```
"cleanup": [
  "/include"
]
```

…to remove development headers as well.

**Comment by play-tarot-2** [comment]
Should I remove the gir / vapi / typelib too?

File: eu.planete_kraus.Tarot.json (hunk 1 above)
Comment (play-tarot-2) [reply eu.planete_kraus.Tarot.json]: Sorry I had not seen this dialog box.  Should I remove the gir / vapi / typelib?  I did not see a mention of them in the doc (http://docs.flatpak.org/en/latest/manifests.html) but obviously they are development files.

Comment (play-tarot-2) [reply eu.planete_kraus.Tarot.json]: OK it works without these.

**Comment by nedrichards** [comment]
bot, build eu.planete_kraus.Tarot

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/eu.planete_kraus.Tarot

**Pull request closed without merging**